docs/datasources/core/instances.md

```diff
@@ -36,6 +36,7 @@ The following attributes are exported:
 * `image_id` - The image used to boot the instance. You can enumerate all available images by calling ListImages.
 * `state` - The current state of the instance: [PROVISIONING, RUNNING, STARTING, STOPPING, STOPPED, CREATING_IMAGE, TERMINATING, TERMINATED]
 * `metadata` - Custom metadata that you provide.
+* `extended_metadata` - Custom nested metadata that you provide.
```

Comment:
Recommend adding a note on how json vs invalid json is handled and represented in the system.